test/integration/test_advective_diffusive.py

```diff
@@ -142,15 +141,13 @@ def __init__(self, g, physics='transport'):
         SourceAdvectiveDiffusiveProblem.__init__(self, g, physics=physics)
 
     def bc(self):
-        dir_faces = self.grid().has_face_tag(FaceTag.DOMAIN_BOUNDARY)
-        dir_faces = np.ravel(np.argwhere(dir_faces))
+        dir_faces = self.grid().get_domain_boundary_faces()
```

Comment:
should this function be removed and replaced by using g.tags['domain_boundary_faces') ??